network/network.go

```diff
@@ -58,4 +59,6 @@ type Network interface {
 
 	// ReceivedSyncMsgChan returns the channel for sync messages
 	ReceivedSyncMsgChan() <-chan *SyncChanObj
+
+	SubscribeTopic(validatorPk *bls.PublicKey) error
```

Comment:
not sure this should be part of an interface.. topics are really implementation specific

Reply:
what you mean is not clear for me, how you suggest to implement that func (or where?)